src/styles/blocks/general.scss

```diff
@@ -0,0 +1,9 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+  font-size: 30px;
+}
+
+h1 {
+  display: none;
+}
```

Comment:
Just remove this ellement from your page, you dont need this element)

Reply:
I don't want to see word Calendar on my screen. This rule is for it.

Reply:
Strange solution, you could just remove that headings 🙂